tests/test_phenopacket_utils.py

```diff
@@ -480,6 +480,22 @@ def test_find_identifier(self):
         self.assertEqual(self.gene_identifier_updater_entrez.find_identifier("A2M"), "2")
         self.assertEqual(self.gene_identifier_updater_entrez.find_identifier("GBA"), "2629")
 
+    def test_obtain_gene_symbol_from_identifier_hgnc(self):
+        self.assertEqual(
+            self.gene_identifier_updater_ens.obtain_gene_symbol_from_identifier(
+                "HGNC:5", "hgnc_id"
```

Comment:
For things like keys in standard dicts, I recommend using constants, because of the likely of change later down the line

HGNC_ID = "hgnc_id" in constants.py

and here:

```suggestion
                "HGNC:5", HGNC_ID
```

Reply:
But feel free to ignore if it blocks your flow or you disagree.

Reply:
I agree, didn't think of that!